### ISSUE TITLE
Inconsistent Attribution

### ISSUE THREAD

**Issue by GuiltyDolphin** [duckduckgo/community-platform#1263]
I've been noticing some inconsistencies in the attribution:

![screenshot from 2016-03-07 17 16 11](https://cloud.githubusercontent.com/assets/8598426/13577060/88a18f6a-e488-11e5-96c3-b6e049dda07e.png)

VS:

![screenshot from 2016-03-07 17 16 19](https://cloud.githubusercontent.com/assets/8598426/13577052/81df8d26-e488-11e5-8d86-6b0ac858a15f.png)

And I swear I saw `Ben Moon` somewhere...

(Just so yea know, it should be `GuiltyDolphin` - and is in the majority of places).

**Comment by MariagraziaAlastra** [comment]
Which IAs are those?

**Comment by GuiltyDolphin** [comment]
@MariagraziaAlastra First is CaesarCipher (I think); second is Regexp.

Also, it looks like that script got implemented! :wink: https://github.com/duckduckgo/community-platform/pull/1256 - we gonna see some mass maintainer updates soon?

**Comment by MariagraziaAlastra** [comment]
Yep, soon :)

**Comment by GuiltyDolphin** [comment]
@MariagraziaAlastra Sweet! Looking forward to it :dolphin:

**Comment by MariagraziaAlastra** [comment]
Looks like in both those IA Pages your name is in camel case, so maybe you just need to wait a bit to see the data updated :)
//cc @jdorweiler do you know anything about this?

Also let me know whenever you find the IAs for which you're given credit as Ben Moon and I'll change the attribution data. Thanks :)

**Comment by GuiltyDolphin** [comment]
@MariagraziaAlastra Will do! Thanks.

**Comment by GuiltyDolphin** [comment]
@MariagraziaAlastra Oh! Another note:

The query https://duckduckgo.com/?q=time+now+%2B+3+hours&t=ffab&ia=answer (DateMath) links me to `duck.co` in the attribution, but most others link to GitHub (there may be others I've missed).

**Comment by MariagraziaAlastra** [comment]
Not really, but maybe we should default to the github account instead (right now it's the other way around). Do you have your github account linked to your duck.co account?

**Comment by MariagraziaAlastra** [comment]
Anyway I changed the attribution on that IA to link to your github account instead.

**Comment by GuiltyDolphin** [comment]
@MariagraziaAlastra Thanks! And yeah - my accounts are linked.
